packages/solutions/app-tools/package.json

```diff
@@ -75,6 +75,7 @@
     "@modern-js/builder-plugin-node-polyfill": "workspace:*",
     "@modern-js/builder-shared": "workspace:*",
     "@modern-js/builder-webpack-provider": "workspace:*",
+    "@modern-js/builder-rspack-provider": "workspace:*",
```

Comment:
`@modern-js/builder-rspack-provider` should not be declared in the `dependencies` field.

Reply:
done